components/ruby/chef-licensing.gemspec

```diff
@@ -12,7 +12,7 @@ Gem::Specification.new do |spec|
   spec.summary       = %q{Chef License storage, generation, and entitlement}
   spec.description   = %q{Ruby library to support CLI tools that use Progress Chef license storage, generation, and entitlement.}
   spec.homepage      = "https://github.com/chef/chef-licensing"
-  spec.required_ruby_version = Gem::Requirement.new(">= 2.3.0")
+  spec.required_ruby_version = Gem::Requirement.new(">= 3.0.3")
```

Comment:
I can see the [system requirements](https://github.com/chef/chef-licensing/tree/main/components/ruby#system-prerequisites) says  Ruby version >= 3.1

Reply:
could we update that as well?

Reply:
Nice catch!